src/server/editionsPage.tsx

```diff
@@ -46,23 +41,31 @@ const styles = `
 
 const renderHead = (
 	request: RenderingRequest,
-	emotionCritical: EmotionCritical,
-): string =>
-	renderToString(
-		<>
-			<meta charSet="utf-8" />
-			<title>{request.content.webTitle}</title>
-			<meta name="viewport" content="initial-scale=1" />
-			<meta
-				httpEquiv="Content-Security-Policy"
-				content={csp([styles, emotionCritical.css])}
-			/>
-			<style data-emotion-css={emotionCritical.ids.join(' ')}>
-				{emotionCritical.css}
-			</style>
-		</>,
+	thirdPartyEmbeds: ThirdPartyEmbeds,
+	itemStyles: string,
+	emotionIds: string[],
+	inlineStyles: boolean,
+): string => {
+	const generalStyles = styles;
+	const cspString = buildCsp(
```

Comment:
Are you sure you want the same CSP as live apps content? This ties them together, and means that the Editions CSP may be more permissive than it has to be, and likewise for the live apps CSP if Editions needs something enabled.

Would it make sense to abstract out some of the logic in the `csp` and `buildCsp` functions and generate two different CSPs using that logic?